### ISSUE TITLE
Description is incorrect when the build is in pending or error state

### ISSUE THREAD

**Issue by ObjColumnist** [bitrise-steplib/steps-github-status#8]
I have tried adding this to my bitrise workflow, the description is set to "The build succeeded" even if it is only pending.

**Comment by viktorbenei** [comment]
Please copy paste this step's config from your `bitrise.yml`

> the description is set to "The build succeeded" even if it is only pending.

That's simply not possible from this step, as if the build is pending that means this step didn't even run at that time, there's nothing what would set that status O_O

**Comment by ObjColumnist** [comment]
I manually send pending so that it updates on Github

```
    - github-status@1.0.3:
        inputs:
        - auth_token: ###
        - set_specific_status: pending
```

But looking at the source code, the description only has 2 possible values:

```
    req.body = {
      state: status,
      target_url: ci_build_url,
      description: (build_is_green ? 'The build succeeded' : 'The build failed. Check the logs on Bitrise'),
      context: 'continuous-integration/bitrise'
    }.to_json
```

**Comment by viktorbenei** [comment]
Ahh, I see the issue. So the status is set correctly (e.g. "pending") it's just the description what's wrong?

**Comment by ObjColumnist** [comment]
<img width="784" alt="pending pull request" src="https://cloud.githubusercontent.com/assets/203059/26723416/55b7e67c-478c-11e7-944e-eacffc85b0c3.png">
Yeah the status works fine, just the incorrect message.

**Comment by viktorbenei** [comment]
Perfect, I think everything's clear now.

Feel free to send us a pull request with a fix, or create a https://discuss.bitrise.io/c/feature-request to bump the priority of this issue!

**Comment by viktorbenei** [comment]
Fixed in v2.0.0 ( https://github.com/bitrise-steplib/steps-github-status/releases/tag/2.0.0 ).

Thanks for reporting @ObjColumnist ! :)
